src/lazy/text/buffer.rs

```diff
@@ -1759,7 +1859,7 @@ mod tests {
         /// contents of the input are considered a complete token.
         fn new(input: &str) -> Self {
             MatchTest {
-                input: format!("{input}\n0"), // add whitespace and a trailing value
```

Comment:
🗺️ Now that we're using `complete_` parsers where necessary, we no longer have to append extra values to our test data.